cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -22,49 +15,79 @@
+    public static final Parameter CONVERT_BOUNDARY_PARAMETER = new Parameter(CONVERT_BOUNDARY, ParameterType.BOOLEAN, "Convert boundary during import", Boolean.FALSE)
             .addAdditionalNames("convertBoundary");
-    private static final Parameter CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE_PARAMETER
+    public static final Parameter CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE_PARAMETER
             = new Parameter(CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE, ParameterType.BOOLEAN, "Create busbar section for every connectivity node", Boolean.FALSE)
             .addAdditionalNames("createBusbarSectionForEveryConnectivityNode");
-    private static final Parameter POWSYBL_TRIPLESTORE_PARAMETER = new Parameter(POWSYBL_TRIPLESTORE, ParameterType.STRING, "The triplestore used during the import", TripleStoreFactory.defaultImplementation())
+    public static final Parameter POWSYBL_TRIPLESTORE_PARAMETER = new Parameter(POWSYBL_TRIPLESTORE, ParameterType.STRING, "The triplestore used during the import", TripleStoreFactory.defaultImplementation())
             .addAdditionalNames("powsyblTripleStore");
-    private static final Parameter STORE_CGMES_MODEL_AS_NETWORK_EXTENSION_PARAMETER = new Parameter(STORE_CGMES_MODEL_AS_NETWORK_EXTENSION, ParameterType.BOOLEAN, "Store the initial CGMES model as a network extension", Boolean.TRUE)
+    public static final Parameter STORE_CGMES_MODEL_AS_NETWORK_EXTENSION_PARAMETER = new Parameter(STORE_CGMES_MODEL_AS_NETWORK_EXTENSION, ParameterType.BOOLEAN, "Store the initial CGMES model as a network extension", Boolean.TRUE)
             .addAdditionalNames("storeCgmesModelAsNetworkExtension");
+    public static final Parameter POST_PROCESSORS_PARAMETER = new Parameter(POST_PROCESSORS, ParameterType.STRING_LIST, "Post processors", Collections.emptyList())
+            .addAdditionalNames("postProcessors");
+
+    private static final List<Parameter> PARAMETERS = Arrays.asList(CHANGE_SIGN_FOR_SHUNT_REACTIVE_POWER_FLOW_INITIAL_STATE_PARAMETER,
+                                                                    CONVERT_BOUNDARY_PARAMETER,
+                                                                    CREATE_BUSBAR_SECTION_FOR_EVERY_CONNECTIVITY_NODE_PARAMETER,
+                                                                    POWSYBL_TRIPLESTORE_PARAMETER,
+                                                                    STORE_CGMES_MODEL_AS_NETWORK_EXTENSION_PARAMETER,
+                                                                    POST_PROCESSORS_PARAMETER);
+
+    private final Map<String, CgmesImportPostProcessor> postProcessors;
 
     private final ParameterDefaultValueConfig defaultValueConfig;
 
-    public CgmesImport(PlatformConfig platformConfig) {
+    public CgmesImport(PlatformConfig platformConfig, List<CgmesImportPostProcessor> postProcessors) {
```

Comment:
Should be a Set<CgmesImportPostProcessor>, but it probably break the API

Reply:
It directly comes from ServiceLoaderCache which return a List, but without any duplicated services.